src/instance/app.cfg.example

```diff
@@ -2,6 +2,17 @@
 APP_CLIENT_ID = ''
 APP_CLIENT_SECRET = ''
 
+# AWS credentials for access such as S3 and presigned URLs
+# https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html
+AWS_ACCESS_KEY_ID = ''
+AWS_SECRET_ACCESS_KEY = ''
+AWS_S3_BUCKET_NAME = ''
+AWS_OBJECT_URL_EXPIRATION_IN_SECS = 60*60 # 1 hour
+# Large response threshold, as determined by len() for the character set, above
+# which responses will be stashed in an S3 bucket and a pre-signed URL
+# returned in the response to avoid the AWS Gateway 10Mb constraint
+LARGE_RESPONSE_THRESHOLD = 9*(2**20)
```

Comment:
Regarding the threshold, 9*(2**20) bytes is 9MB, should it be 10MB?

Reply:
I was just trying to stay below the known threshold, but not too far below.  I also thought the exponential notation of megabytes was clearer for what needed to be done, but didn't want to induce a non-integer by multiplying by 9.9 or something.  If we want to be closer, maybe we specify a number directly, like 10485759.  But appreciating [the multibyte character consideration you mention in the other comment](https://github.com/hubmapconsortium/uuid-api/pull/107#discussion_r941438115), I'd like to have a little more headroom before reaching the AWS Gateway limit.